dbt/adapters/dremio/connections.py

```diff
@@ -258,12 +258,10 @@ def execute(
     def drop_catalog(self, database, schema):
         logger.debug('Dropping schema "{}.{}"', database, schema)
 
-        connection = self.get_thread_connection()
+        connection_param = self.get_thread_connection()
+        connection = self.open(connection_param)
         credentials = connection.credentials
-        api_parameters = self.build_api_parameters(credentials)
-
-        token = login(api_parameters)
-        connection.credentials.token = token
+        api_parameters = connection.handle._parameters
```

Comment:
Can we keep things consistent? Either have connection_parameters and api_parameters or connection_param and api_param.

Reply:
What type does `self.get_thread_connection()` return? 
And let's use the full word if it is "parameters".

Reply:
In Python, anything starting with an underscore is considered private. So we shouldn't call `connection.handle.**_**parameters`. 

I believe i created that private member variable. I don't think there is any reason to make it private, but let's think about if there would be any consequences of doing so (unlikely).

If we decide we are ok with sharing that info, then add a getter method.

Reply:
How about just "connection"?